src/mixins/common-mixin.ts

```diff
@@ -7,15 +7,23 @@ import { omit, isEqual } from 'lodash'
 @Component({})
 export default class CommonMixin extends Vue {
   /**
-   * Compares two objects while omitting specified properties from the comparison.
+   * Compares two objects while omitting a specified property from the comparison.
    *
-   * @param addressA The first object to compare
-   * @param addressB The second object to compare
-   * @param prop The property to omit during the comparison
+   * @param addressA the first object to compare
+   * @param addressB the second object to compare
+   * @param prop the property to omit during the comparison
```

Comment:
I believe lower case is what is expected here -- if you hover over the method name then you get the JSDoc for it and it looks right in lower case (and wrong in upper case).